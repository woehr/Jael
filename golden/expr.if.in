if x then y else { z = 1; z }