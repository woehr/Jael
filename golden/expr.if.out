JEIf
  (JELIdent "x")
  (JELIdent "y")
  (JELet
     [_Ã—_ "z" (JEInt IntInfo {intDigits = 1, intValue = 1})]
     (JELIdent "z"))
